jobs/eqterraform.groovy

```diff
@@ -7,9 +7,9 @@ job('Terraform Environment') {
        githubPush()
   }
   steps {
-    shell('/opt/go/bin/terraform remote config -backend=S3 -backend-config="bucket=jenkins-ci-production-terraform-state" -backend-config="key=jenkins" -backend-config="region=eu-west-1"')
-    shell('/opt/go/bin/terraform remote pull')
-    shell('/opt/go/bin/terraform apply -var "env=prod"')
-    shell('/opt/go/bin/terraform remote push')
+    shell('cd survey-runner; terraform remote config -backend=S3 -backend-config="bucket=jenkins-ci-production-terraform-state" -backend-config="key=jenkins" -backend-config="region=eu-west-1"')
```

Comment:
Will this overwrite the survey runner remote config? I'm not sure how to test this short of looking in the bucket at the statefile?